frontend/src/components/ProfileContent.tsx

```diff
@@ -51,6 +53,13 @@ const useStyles = makeStyles({
     padding: 5,
     paddingRight: 10,
   },
+  recordButton: {
+    backgroundColor: Colors.DEEP_BLUE,
+    color: "white",
+    "&:hover": {
+      backgroundColor: Colors.DEEP_RED,
+    },
+  },
```

Comment:
This is in keeping with what we already have by way of buttons? Or what we plan to have?

Reply:
We have a task planned to make the css coloring system consistent, using themes. For now, this is fine as long as it works